logging/resource-logs-viewing.adoc

```diff
@@ -0,0 +1,23 @@
+:context: resource-logs-viewing
+[id="resource-logs-viewing"]
+= Viewing logs for a resource
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+You can view the logs for various resources, such as builds, build configs, deployment configs, and pods by using the {product-title} CLI and the {product-title} web console. 
+
+[NOTE]
+====
+Resource logs are a default feature that provides limited log viewing capability. To enhance your log retrieving and viewing experience, it is recommended that you install xref:../logging/cluster-logging.adoc#cluster-logging[{product-title} cluster logging], which aggregates all the logs from your {product-title} cluster, such as node system audit logs, application container logs, and infrastructure logs, into a dedicated log store.  You can then query, discover, and visualize your log data through the xref:../logging/cluster-logging-visualizer.adoc#cluster-logging-visualizer-using[Kibana interface]. Resource logs do not access the cluster logging log store. 
+====
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/resource-logs-viewing-cli-console.adoc[leveloffset=+1]
+////
+include::modules/resource-logs-viewing-console.adoc[leveloffset=+1]
```

Comment:
Yeah, it is commented-out here. Eric Rich suggested combining the console and CLI steps into one module. I left the link to the console-only module in case a peer-reviewer thought it violated the mod docs formula. I removed it.

Reply:
I was considering also suggesting that this be split into two different procedure modules. To me, the procedure is viewing logs for a resource. There are two ways to do so, via the CLI or the web console.